cocotb/binary.py

```diff
@@ -127,6 +130,15 @@ def __init__(self, value=None, bits=None, bigEndian=True,
         if value is not None:
             self.assign(value)
 
+        # bits is the deprecated name for n_bits, allow its use for
+        # backward-compat reasons.
+        if (bits is not None and n_bits is not None):
+            raise ValueError("You cannot use n_bits and bits at the same time.")
+        if bits is not None:
+            n_bits = bits
```

Comment:
Since this  is a deprecation, you should add
```python
warnings.warn(
    "The bits argument to BinaryValue has been renamed to n_bits",
    DeprecationWarning, stacklevel=2)
```